### DISCUSSION TITLE
Notification dot over profile icon?

### DISCUSSION THREAD

**Discussion by OverLordGoldDragon** [community/community#23636] [New to GitHub]
<p>I have my usual notification dot over the bell icon, and today for the first time it’s over my profile:</p>
<p><img src="https://user-images.githubusercontent.com/16495490/181078588-037f876d-89be-4678-847a-a1eba69a2355.png" alt="image" data-base62-sha1="slLBihKUlt79rZR5QL90wA7A5jC" width="160" height="66"></p>
<p>What is it? Clicking doesn’t make it go away.</p>

**Comment by rezk2ll** [comment, marked as answer]
<p>it’s a feature preview notification.</p>

**Comment by zgoda** [comment]
<p>Gone away once I visited account upgrade page.</p>

**Comment by lostpebble** [comment]
<p>Come on, this is like UX 101. A notification dot without any way for us to know what it causing it? OCD nightmare (even for mild cases).</p>

**Comment by OverLordGoldDragon** [comment]
<p>A dot should appear next to it then - “Feature preview” is always there.</p>
